Monika After Story/game/script-grammar.rpy

```diff
@@ -0,0 +1,375 @@
+    m 1eua "Only one replacement should make sense, and that should tell you which one to use!"
+    m 3eua "Let's take, for example, the title of my poem, {i}The Lady who Knows Everything{/i}."
+    m 3esd "If we just look at the clause '{b}who knows everything{/b}' and replace the '{b}who{/b},' we get..."
+    m 1esd "'{b}She knows everything{/b}' or '{b}her knows everything{/b}.'"
+    m 3euc "Only '{b}she knows everything{/b}' makes sense, so the correct phrase is '{b}who knows everything{/b}.'"
+    m 1hksdla "Who said writing was hard?"
+    m 1eub "That's all I have for today, [player]."
+    m 3hub "Thanks for listening!"
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_gtod_tip008",
+            category=["grammar tips"],
+            prompt="And I vs. And me",
+            conditional="store.mas_gtod.has_day_past_tip(7)",
+            action=EV_ACT_POOL
+        )
+    )
+
+label monika_gtod_tip008:
+    m 1eua "Last time we talked about the difference between '{b}who{/b}' and '{b}whom{/b}.'"
+    m 1esd "Another couple of words that can be just as confusing to use are '{b}and I{/b}' and '{b}and me{/b}.'"
+    m 3etc "Is it '{b}[player] and I went on a date{/b}' or '{b}[player] and me went on a date{/b}?'"
+    m 3eud "Just like with '{b}who{/b}' and '{b}whom{/b},' the issue boils down to one of subjects and objects."
+    m 3eub "And just like when we talked about '{b}who{/b}' versus '{b}whom{/b},' it turns out there's a simple way to figure out which one is correct!"
+    m 1euc "In our example, if you just take out '{b}[player] and{/b}' from the sentence, only one should make sense."
+    m 1hua "Let's try it out!"
+    m 3eud "The end result is: {w=0.5}'{b}I went on a date{/b}' or '{b}me went on a date{/b}.'"
+    m 3eub "Clearly, only the first one makes sense, so it's '{b}[player] and I went on a date{/b}.'"
+    m 1tuu "Oh, sorry, [player]...{w=1}did it make you feel left out when I said only '{b}I went on a date{/b}?'"
+    m 1hksdlb "Ahaha! Don't worry, I'd never leave you behind."
+    m 1eua "Anyways, that's all for today, [player]."
+    m 3hub "Thanks for listening!"
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_gtod_tip009",
+            category=["grammar tips"],
+            prompt="Apostrophes",
+            conditional="store.mas_gtod.has_day_past_tip(8)",
+            action=EV_ACT_POOL
+        )
+    )
+
+# Note: formatted apostrophes have been used in examples in this tip for clarity. Please DO NOT remove them.
+label monika_gtod_tip009:
+    if player[-1].lower() == 's':
+        $ tempname = player
+    else:
+        $ tempname = 'Alexis'
+
+    m 1eua "Today we're going to talk about apostrophes. Pretty straight forward, right?"
```

Comment:
```suggestion
    m 1eua "Today we're going to talk about apostrophes. Pretty straightforward, right?"
```

While "straight forward" is not _necessarily_ wrong, "straightforward" is more commonly used.